packages/access-token-jwt/src/jwt-verifier.ts

```diff
@@ -165,7 +220,7 @@ const jwtVerifier = ({
 
   return async (jwt: string) => {
     try {
-      if (!jwksUri) {
+      if (issuerBaseURL) {
```

Comment:
If you provide an `issuerBaseURL` (you're doing discovery) and an `issuer` - currently the discovered issuer takes precedence (you can configure the `iss` validation using `validators` option).

Q. Should I error if you provide both? Since `issuer` will effectively be ignored in this case. Or should `issuer` take precendece and I ignore the discovered issuer? Or is it ok as is?